traveler-ios-ui/TravelerKitUI/UIUtilities/FormView/FormView.swift

```diff
@@ -108,11 +124,24 @@ extension FormView: UICollectionViewDataSource {
     }
 
     public func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return dataSource!.formView(self, numberOfFieldsIn: section)
+        let count = dataSource!.formView(self, numberOfFieldsIn: section)
+        return datePickerIndexPath?.section == section ? count + 1 : count
     }
 
     public func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        let descriptor = dataSource!.formView(self, inputDescriptorForFieldAt: indexPath)
+        guard indexPath != datePickerIndexPath else {
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: datePickerCellIdentifier, for: indexPath) as! FormDateInputCell
+            let valueCellIndexPath = IndexPath(item: indexPath.row - 1, section: indexPath.section)
+            cell.delegate = self
+            cell.datePicker.minimumDate = nil // TODO
+            cell.datePicker.maximumDate = nil // TODO
```

Comment:
do the min and max date values just need to be added or is this something more involved?

Reply:
we have to wait for the API to implement it then add it to our models

Reply:
ok sweet mind adding that to the comment 👍

Reply:
done 👍